src/Bau/Arguments.cs

```diff
@@ -14,17 +14,20 @@ public class Arguments
     {
         private readonly ReadOnlyCollection<string> tasks;
         private readonly LogLevel logLevel;
+        private readonly TaskListWriter taskListWriter;

```

Comment:
- [ ] This is pretty poopie, will fix.

Probably going to store some kind of enumeration there for the task listing type that can be used in a factory of some sort within `Run()` to create the TaskListWriter.